runelite-client/src/main/java/net/runelite/client/plugins/combatcounter/CombatCounter.java

```diff
@@ -114,7 +116,7 @@ CombatCounterConfig provideConfig(ConfigManager configManager)
 		return configManager.getConfig(CombatCounterConfig.class);
 	}
 
-	private Map<Integer, Integer> variables = new HashMap<Integer, Integer>()
+	private final Map<Integer, Integer> VARIABLES = new HashMap<Integer, Integer>()
```

Comment:
This looks like it should be a static final ImmutableMap (and the data in it looks like it should go into api instead of here)